xarray/tests/test_dataarray.py

```diff
@@ -6178,6 +6178,8 @@ def test_rolling_iter(da):
                 actual.values[actual.values.nonzero()],
                 expected.values[expected.values.nonzero()],
             )
+    # no error
+    repr(rolling_obj)
```

Comment:
could you move this to its own test? That way this test cannot be shadowed by a failure in the code above